### DISCUSSION TITLE
merge Signac objects and choose the peak features?

### DISCUSSION THREAD

**Discussion by hongruhu** [stuart-lab/signac#1617] [General]
Hi, I was wondering when people are merging Signac objects based on different libraries/samples, which of the following would you recommend to find the unified peak feature set (and why):

1. merge the fragments files from all samples first followed by sorting and indexing, re-calling peaks using the new fragments file.
2. do what's described in https://stuartlab.org/signac/articles/merging (create common peak sets using the cellranger output fragment files from each individual sample) 

@timoast appreciate it for your time in advance!

**Comment by timoast** [comment]
We recommend doing what's shown in the Signac merge vignette

Merging the fragment files is very time consuming and creates duplicated files, and required changing the cell barcode in each file so there is no overlap. We designed Signac to avoid needing to do this.
